pkg/apis/sources/v1alpha1/mongoDbSource_defaults.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2019 The Knative Authors
```

Comment:
wrong year? Can you update to 2020 all of the files?

Reply:
also the file names should be all lowercase... mongodbsource_defaults.go.
Same for others

Reply:
also check the copyright is Google LLC.
you can see that in hack/boilerplate I think